parser/tests/requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+neo4j=='4.4.5'
+pytest=='7.1.2'
+re=='2.2.1'
+Unidecode=='1.3.4'
```

Comment:
```suggestion
attrs==22.1.0
iniconfig==1.1.1
neo4j==4.4.5
packaging==21.3
pluggy==1.0.0
py==1.11.0
pyparsing==3.0.9
pytest==7.1.2
pytz==2022.1
tomli==2.0.1
Unidecode==1.3.4
```

Reply:
Oh thank you ! I didn't know how to do it